scripts/restore

```diff
@@ -55,14 +49,16 @@ sudo chown -R root:root $final_path
 sudo chown -R www-data:root $final_path/{data,tmp}
 sudo chmod -R 700 $final_path/{data,tmp}
 
-# Restore conf files
-sudo cp -a "./conf/nginx.conf" $conf
-
-# Reload Nginx
-sudo service nginx reload
+# Restore configuration files
+sudo cp -a ./conf/nginx.conf "${nginx_conf}"
 
 # Set ssowat config
+ynh_app_setting_set "$app" is_public "$is_public"
```

Comment:
in fact is_public is yes is not relevant ? so prefer is_public is no ?
```
# Set ssowat config
if [ "$is_public" = "No" ];
then
	ynh_app_setting_delete $app skipped_uris
fi
```

Thanks